modules/appeals_api/appeals_api.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |s|
   s.files = Dir['{app,config,db,lib}/**/*', 'MIT-LICENSE', 'Rakefile', 'README.rdoc']
   s.test_files = Dir['spec/**/*']
 
-  s.add_dependency 'rails', '~> 5.2.3'
+  s.add_dependency 'rails', '>= 5.2.3'
```

Comment:
I _feel_ like all these module `rails` depencies should just specify `'~> 6.0.2'` since they're not going to be able to use any deprecated/removed Rails 5 features anyway.  Am I being nitpicky? (Or am I wrong?)